python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -156,3 +154,41 @@ def compile(model: nn.Module,
                 return pl_model
         else:
             return pl_model
+
+    def quantize(self, model, calib_dataloader, val_dataloader=None, metric: str = None,
+                 backend='inc', conf=None, framework='pytorch_fx', approach='ptsq',
+                 strategy='bayesian', accuracy_criterion=None, timeout=0, max_trials=1):
+        if backend == 'inc':
+            from bigdl.nano.quantization import QuantizationINC
+            quantizer = QuantizationINC(framework=framework, conf=conf, approach=approach,
+                                        strategy=strategy, accuracy_criterion=accuracy_criterion,
+                                        timeout=timeout, max_trials=max_trials)
+            q_litmodel = copy.deepcopy(model)
+            quantizer.model = q_litmodel.model
+            q_approach = quantizer.cfg.quantization.approach
+
+            def eval_func(model_to_eval):
+                if val_dataloader:
+                    q_litmodel.model = model_to_eval
+                    val_outputs = self.validate(q_litmodel, val_dataloader)
+                    return val_outputs[0][f'val/{metric}']
+                else:
+                    return 0
+
+            quantizer.eval_func = eval_func
+
+            if q_approach == 'quant_aware_training':
+                def q_func(model_to_train):
+                    q_litmodel.model = model_to_train
```

Comment:
why `q_litmodel.model`?

Reply:
The same as above, make `model_to_train` as a PL Module, and call 'fit(...)' for QAT.